code/Serving_API.py

```diff
@@ -180,10 +257,27 @@ def physical_route(src_latitude: float, src_longitude: float,
 
 def run():
     app.db_file = '../database/igdb.db'
-    app.coord_city_map, app.coord_set, app.G = build_up_global_graph(app.db_file)
+    app.coord_city_map, app.coord_set, app.G, app.asn_nodes = build_up_global_graph(
+        app.db_file)
     import uvicorn
     uvicorn.run(app, port=8082)
 
 
+def find_closest_paths(lat: float, lon: float, db_path: str, max_distance: float) -> gpd.GeoDataFrame:
```

Comment:
It doesn't look like you'll need the full GeoDataFrame from the caller. So why not just return a boolean? Since that's all the caller wants.

And this function can be simplified to be something like `is_point_close_to_path(...) -> bool`?

Reply:
fixed to is_point_close_to_path(...) -> bool